src/clientagent/AstronClient.cpp

```diff
@@ -375,10 +407,12 @@ class AstronClient : public Client, public NetworkClient
             NetworkClient::send_disconnect();
         }
         break;
-        case CLIENT_OBJECT_SET_FIELD: {
```

Comment:
This change may or may not be useful, but I don't see why it belongs in this pull request. Am I missing something?